DESCRIPTION

```diff
@@ -55,6 +55,7 @@ Imports:
     backports,
     cli (>= 2.0.0),
     crayon,
+    desc, 
```

Comment:
Can we suggest? Are the other imports really necessary too?

Reply:
At first I thought that we cannot put it into suggests because the call is not within a step but only within the macro and hence won't be autoinstalled when just adding `do_drat()` by the `$prepare` method.
That's why I put it into imports.

However, if I move the check into `step_add_to_drat()` it should work. We stop a bit later but it saves us the IMPORTS listing. We just do not build the binaries then and then there also shouldn't be anything to deploy.